common/rya.api/src/main/java/mvm/rya/api/instance/RyaDetailsToConfiguration.java

```diff
@@ -0,0 +1,66 @@
+package mvm.rya.api.instance;
+
+import java.util.Map;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.log4j.Logger;
+
+import com.google.common.base.Optional;
+import com.google.common.base.Preconditions;
+
+import mvm.rya.api.instance.RyaDetails.PCJIndexDetails;
+import mvm.rya.api.instance.RyaDetails.PCJIndexDetails.PCJDetails;
+
+/**
+ * Used to fetch {@link RyaDetails} from a {@link RyaDetailsRepository} and
+ * add them to the application's {@link Configuration}.
+ */
+public class RyaDetailsToConfiguration {
+    private static final Logger LOG = Logger.getLogger(RyaDetailsToConfiguration.class);
+    public static final String CONF_PCJ_IDS = "pcj.index.ids";
```

Comment:
Just kidding, this isn't what I thoguht it was. I think we can delete this entirely for this review.